jest.config.js

```diff
@@ -1,5 +1,11 @@
 "use strict";
+
+const installPrettier = require("./scripts/install-prettier");
+
 const ENABLE_CODE_COVERAGE = !!process.env.ENABLE_CODE_COVERAGE;
+if (process.env.NODE_ENV === "production") {
```

Comment:
I don't think use `NODE_ENV` environment is good idea, maybe change to another variable  later